### DISCUSSION TITLE
inpu maskked number

### DISCUSSION THREAD

**Discussion by maneljove** [quasarframework/quasar#11731] [General - Components / Directives / etc]
Code: 

<div id="q-app" style="min-height: 100vh;">
  <div class="q-pa-md" style="max-width: 300px">
    <div class="q-gutter-md">
            <q-input v-model.number="quantity" mask="###" >
                  <template v-slot:prepend>
                       <q-btn flat icon="remove" size="xs"  :disable="!(quantity>0)" @click="quantity -= 1"/>
                  </template>
                  <template v-slot:append>
                       <q-btn flat icon="add" size="xs" :disable="quantity > stock" @click="quantity += 1"/>
                  </template>
            </q-input>
    </div>
  </div>
</div>

value not refreshing when clicking buttons but update model

**Comment by MilosPaunovic** [comment]
As the [documentation](https://quasar.dev/vue-components/input#mask) states, `Mask is only available if the type is one of ‘text’ (default), ‘search’, ‘url’, ‘tel’, or ‘password’.`, see `Warning` card.

**Comment by maneljove** [comment]
Tjhank you!
________________________________
De: Milos Paunovic ***@***.***>
Enviat el: diumenge, 19 de desembre de 2021 14:52
Per a: quasarframework/quasar ***@***.***>
A/c: Manel Jove ***@***.***>; Author ***@***.***>
Tema: Re: [quasarframework/quasar] inpu maskked number (Discussion #11731)


As the documentation<https://quasar.dev/vue-components/input#mask> states, Mask is only available if the type is one of ‘text’ (default), ‘search’, ‘url’, ‘tel’, or ‘password’., see Warning card.

—
Reply to this email directly, view it on GitHub<https://github.com/quasarframework/quasar/discussions/11731#discussioncomment-1843272>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AW7KIKLDMAOQYFYO7AWBGHTURXPSLANCNFSM5KL4RSXA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
